pages/common/vinmap.md

```diff
@@ -0,0 +1,24 @@
+# vinmap
+
+> A multithreaded Nmap scanner that splits IP ranges into chunks, performs parallel scans, and merges XML or JSON results. **Version:** 1.0.0.
+> More information: <https://pypi.org/project/vinmap>.
+
+- Perform a basic scan of a subnet:
+
+`vinmap -ip {{192.168.1.0/24}}`
+
+- Scan a domain with version and OS detection, saving results to a specific file:
+
+`vinmap -ip {{example.com}} -s "-sV -O" -o {{scan_results.xml}}`
```

Comment:
So just as an fyi for this, I confirmed the change, and it does let you put a path, but it also copies the results to ~/NMAP/{xml_filename} so it can be opened with zenmap on all OS' without needing manual intervention